CrossPlatformDesktopProject/CrossPlatformDesktopProject/Game1.cs

```diff
@@ -95,6 +95,11 @@ protected override void UnloadContent()
         protected override void Update(GameTime gameTime)
         {
             currentState.Update();
+
+            if (player.link_health == 6)
```

Comment:
It might be better to have this be player.link_health == 0 and have Link's health be a positive value (i.e. 3), then have health removed rather than "added" when he is hit. This works, but it might be clearer if it's the other way around